cmd/jb/install_test.go

```diff
@@ -30,7 +29,7 @@ import (
 func TestInstallCommand(t *testing.T) {
 	testcases := []struct {
 		Name                    string
-		URLs                    []*url.URL
+		URLs                    []string
```

Comment:
nit: this changed semantically to `Paths` now, no?

Reply:
Indeed, this might not yet have been updated in the test :slightly_smiling_face: